apps/editors/templates/editors/includes/history.html

```diff
@@ -3,9 +3,15 @@
     {{ record.log.short }}
   </th>
   <td>
-    {% trans user = record.user|user_link, date = record.created|babel_datetime %}
-      <div>By {{ user }} on {{ date }}</div>
-    {% endtrans %}
+    {% if record.user %}
```

Comment:
From what I've seen, creating records without a user is already done in some places, but i'd rather use the 'Mozilla' account, can you tell me more about it? Is it a user created on production? What's its id/username/email?